src/main/java/net/greatstart/mappers/ProjectMapper.java

```diff
@@ -1,12 +1,28 @@
 package net.greatstart.mappers;
 
+import net.greatstart.dto.DtoInvestment;
 import net.greatstart.dto.DtoProject;
+import net.greatstart.model.Investment;
 import net.greatstart.model.Project;
+import org.mapstruct.Context;
 import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+import java.util.List;
 
 @Mapper(componentModel = "spring")
 public interface ProjectMapper {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1214)